### PR TITLE
fix excluding additional items with the exclude-from-backup label

### LINKED ISSUE #1835: exclude-from-backup not working for PersistentVolume
**What steps did you take and what happened:**
Exclude a PersistentVolume to backup

Step 1:
```
kubectl  label -n kube-tools pv/d-j6cai1gtm341t3ur48v2 velero.io/exclude-from-backup=true
```

Step 2:
```
velero backup  create test
```

Step 3:
```
velero backup describe test --details 
```

Output
```
...
  v1/PersistentVolume:
    - d-j6cai1gtm341t3ur48v2
...
Persistent Volumes:
  d-j6cai1gtm341t3ur48v2:
    Snapshot ID:        s-j6cfjrz9szlt3kmit2s6
    Type:               cloud_ssd
    Availability Zone:  cn-hongkong-b
    IOPS:               <N/A>
...
```

**What did you expect to happen:**

Should no snapshoot taken for the PV

**Environment:**

- Velero version (use `velero version`): 
```
Client:
	Version: v1.1.0
	Git commit: a357f21aec6b39a8244dd23e469cc4519f1fe608
Server:
	Version: v1.1.0

```
- Kubernetes version (use `kubectl version`):
```
Client Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.3", GitCommit:"2d3c76f9091b6bec110a5e63777c332469e0cba2", GitTreeState:"clean", BuildDate:"2019-08-19T12:36:28Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"12+", GitVersion:"v1.12.6-aliyun.1", GitCommit:"8cb561c", GitTreeState:"", BuildDate:"2019-04-22T11:34:20Z", GoVersion:"go1.10.8", Compiler:"gc", Platform:"linux/amd64"}
```
- Cloud provider or hardware configuration:
Alicloud

### PR DESCRIPTION
**Pull request by prydonius**
fixes #1835

Signed-off-by: Adnan Abdulhussein <aadnan@vmware.com>

### CODE REVIEW

File: pkg/backup/backup_test.go [hunk 1]
```diff
@@ -1329,6 +1329,36 @@ func TestBackupActionAdditionalItems(t *testing.T) {
 				"resources/pods/namespaces/ns-2/pod-2.json",
 			},
 		},
+		{
+			name:   "additional items with the velero.io/exclude-from-backup label that are non-included resources are not backed up",
```
Comment (skriss) [inline pkg/backup/backup_test.go]: nit: I'd probably drop "that are non-included resources"

File: pkg/backup/item_backupper.go [hunk 2]
```diff
@@ -123,6 +123,11 @@ func (ib *defaultItemBackupper) backupItem(logger logrus.FieldLogger, obj runtim
 		log = log.WithField("namespace", namespace)
 	}
 
+	if exclude := metadata.GetLabels()["velero.io/exclude-from-backup"]; exclude == "true" {
```
Comment (skriss) [inline pkg/backup/item_backupper.go]: don't really need to declare `exclude` here, can just say `if metadata.GetLabels()["velero.io/exclude-from-backup"] == "true" {`

**Comment by prydonius** [comment]
thanks @skriss, addressed your comments!

**Comment by prydonius** [comment]
@nrb @carlisia PTAL

**Pull request merged**